tools/check_test.js

```diff
@@ -0,0 +1,170 @@
+/* Copyright 2016 Samsung Electronics Co., Ltd.
```

Comment:
If we have to copy test files into IoT devices, I think all of the test driver files should be placed inside `test` directory, not separated in `test` and `tools` each.